lib/path/include/path/Pose.hpp

```diff
@@ -64,6 +64,18 @@ class Pose : public cogip_defs::Pose {
         PB_PathPose &path_pose    ///< [out] Protobuf message to fill
         ) const;
 
+    /// Override operator ==
+    bool operator==(const Pose& other) {
+        return (
+            this->x() == other.x() &&
```

Comment:
The `==` comparison between doubles can cause problems in this case? 
Shouldn't we introduce a tolerance ratio here to ensure good double comparison?

Reply:
indeed, we could define a tolerance for float comparison like:

```C
fabs(this->x()-other.x()) < tolerance;
```

where `tolerance = 1.0e-2`;

Reply:
done